test/Conversion/ATenToXTen/aten_to_xten_resnet_torchvision.mlir

```diff
@@ -0,0 +1,279 @@
+//===- aten_resnet50_torchvision.mlir ---------------------------------*- MLIR -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2022 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
+
+// ResNet50 from Torchvision, converted to MLIR via torch-mlir (op decomposition disabled)
+// Additionally, I added a softmax layer to the end of the model.
```

Comment:
It would be useful if you added the python code to our `sandbox/helloworld` examples.

Reply:
I will add this in a new PR to `vitis_flexml`, I've deleted this mlir file now since we discussed it won't be useful